src/picongpu/submit/hypnos-hzdr/k80_profileRestart.tpl

```diff
@@ -0,0 +1,136 @@
+# send me mails on job (b)egin, (e)nd, (a)bortion or (n)o mail
+#PBS -m !TBG_mailSettings -M !TBG_mailAddress
+#PBS -d !TBG_dstPath
+#PBS -n
+
+#PBS -o stdout
+#PBS -e stderr
+
+echo 'Running program...'
+
+cd !TBG_dstPath
+
+export MODULES_NO_OUTPUT=1
+source ~/picongpu.profile
+if [ $? -ne 0 ] ; then
+  echo "Error: ~/picongpu.profile not found!"
+  exit 1
+fi
+unset MODULES_NO_OUTPUT
+
+#set user rights to u=rwx;g=r-x;o=---
+umask 0027
+
+mkdir simOutput 2> /dev/null
+cd simOutput
+
+
+sleep 1
+
+echo "----- automated restart routine -----"
+
+#check whether last checkpoint is valid
+#export before variable needed to avoid bug #695
+export file=""
+for file in `ls -t ./checkpoints/checkpoint_*.h5`
+do
+    echo -n "validate checkpoint $file: "
+    h5ls $file &> /dev/null
+    if [ $? -eq 0 ]
+    then
+        echo "OK"
+        break
+    else
+        echo "FAILED"
+        export file=""
+    fi
+done
+
+#this sed call extracts the final simulation step from the cfg (assuming a standard cfg) 
+export finalStep=`echo !TBG_programParams | sed 's/.*-s[[:blank:]]\+\([0-9]\+[^\s]\).*/\1/'`
+echo "final step      = " $finalStep
+#this sed call extracts the -s and --checkpoint flags
+export programParams=`echo !TBG_programParams | sed 's/-s[[:blank:]]\+[0-9]\+[^\s]//g' | sed 's/--checkpoints[[:blank:]]\+[0-9]\+[^\s]//g'`
+#extract restart period
+export restartPeriod=`echo !TBG_programParams | sed 's/.*--checkpoints[[:blank:]]\+\([0-9]\+[^\s]\).*/\1/'`
+echo  "restart period = " $restartPeriod
+
+if [ "" != "$file" ]
+then
+    export cptimestep=`basename $file | sed 's/checkpoint_//g' | sed 's/.h5//g'`
+    echo "start time      = " $cptimestep
+
+    export endTime="$(($cptimestep + $restartPeriod ))"
+    echo "end time        = " $endTime
+
+    export stepSetup=$(echo -s $endTime "--restart --restart-step" $cptimestep "--checkpoints" $restartPeriod )
+else
+    echo "no checkpoint found"
+    export endTime=$restartPeriod
+    export stepSetup=$(echo " -s " $endTime "--checkpoints" $restartPeriod )
+fi
+
+echo "-------------------------------------"
+
+#wait that all nodes see ouput folder
+sleep 1
+
+mpiexec --prefix $MPIHOME -tag-output --display-map -x LIBRARY_PATH -x LD_LIBRARY_PATH -am !TBG_dstPath/tbg/openib.conf --mca mpi_leave_pinned 0 -npernode !TBG_gpusPerNode -n !TBG_tasks !TBG_dstPath/picongpu/bin/cuda_memtest.sh
+
+if [ $? -eq 0 ] ; then
+  mpiexec --prefix $MPIHOME -x LIBRARY_PATH -x LD_LIBRARY_PATH -tag-output --display-map -am !TBG_dstPath/tbg/openib.conf --mca mpi_leave_pinned 0 -npernode !TBG_gpusPerNode -n !TBG_tasks !TBG_dstPath/picongpu/bin/picongpu $stepSetup !TBG_author $programParams | tee output
+fi
+
+mpiexec --prefix $MPIHOME -x LIBRARY_PATH -x LD_LIBRARY_PATH -npernode !TBG_gpusPerNode -n !TBG_tasks killall -9 picongpu 2>/dev/null || true
+
+if [ $endTime -lt $finalStep ]
+then
+    echo "submitting followup job"
+    ssh hypnos4 "/opt/torque/bin/qsub !TBG_dstPath/tbg/submit.start"
```

Comment:
- please add after the following after this call.
- maybe remove the `echo ` before the ssh call to reduce the messages

```BASH
if [ $? -ne 0 ] ; then
    echo "error on job submitting"
else
   echo "jib submitted`
fi